pycbc/io/hdf.py

```diff
@@ -648,6 +648,15 @@ def get_snglfile_array_dict(self, variable):
             return_dict[ifo] = curr
         return return_dict
 
+    def ignore_x_loudest(self, num_ignore):
```

Comment:
If you do the former change, correct me if I'm wrong, but you don't need this function anymore.